local/policy/app.yml

```diff
@@ -1,4 +1,4 @@
-# Top-level policy file
+# Root-level policy file
```

Comment:
These are not the root-level policy files though, right? They're actually the leaves on the policy tree. Same with the remote policy files. Should we just remove these comments?

Reply:
True... 😔 I will remove them since they are clearly not clarifying.